sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/Utility.java

```diff
@@ -25,4 +35,64 @@ public static void inputDocumentsValidation(Iterable<?> documents) {
             throw new IllegalArgumentException("'documents' cannot be empty.");
         }
     }
+
+    /**
+     * Get a mock {@link HttpResponse} that only return status code 400.
+     *
+     * @param response A {@link SimpleResponse} with any type
+     * @return A mock {@link HttpResponse} that only return status code 400.
+     */
+    public static HttpResponse getMockHttpResponse(SimpleResponse<?> response) {
```

Comment:
consider changing to a name that indicates that this method is only intended to use when error.id=""